setsofdata.py

```diff
@@ -63,14 +105,76 @@ def create_clinic_services(clinic_spec_data_frame):
             service_name_short_first = "".join([word[0] for word in service_name_first.split()]).upper() 
             service_name_short_second = "".join([word[0] for word in service_name_second.split()]).upper() 
 
-            all_services = [[service_name_first,service_name_short_first],[service_name_second,service_name_short_second]]
-            all_service_df = pd.DataFrame(all_services, columns=['ServiceName','ServiceShort'])
+            base_cost = 100
+            random_price_part = random.choice([100,120])
+
+            service_price_first = (base_cost + random_price_part) * 1
+            service_price_second = (base_cost + random_price_part) * 1.25
+         
+
+            service_cost_first = random_price_part * 1
+            service_cost_second = random_price_part * 1.25
+   
+            base_duration = 15
+
+            service_duration_first = base_duration
+            service_duration_second = base_duration
+
+            all_services = [\
+            [service_name_first,service_name_short_first,service_price_first,service_cost_first,service_duration_first]\
+            ,[service_name_second,service_name_short_second,service_price_second,service_cost_second,service_duration_second]]
+
+            all_service_df = pd.DataFrame(all_services, columns=['ServiceName','ServiceShort','ServicePrice','ServiceCost','ServiceDuration'])
             concated_services = pd.concat([concated_services,all_service_df])
     return concated_services
 
-print(create_clinic_services(ccs))
+def create_clinic_physician(clinic_spec_data_frame):
+    df = clinic_spec_data_frame
+    concated_physician = pd.DataFrame(columns=['FirstName','LastName','LicenseNo','HireDate','Age','Specialty'])
+
+
+    for i,row in df.iterrows():
+        FirstName = ''
+        LastName = names.get_last_name()
+        LicenseNo = "S" + str(row['Specialty'])[0] + str(random.randint(10000,99999))
+        HireDate = date.today()
+        Age = random.randint(27,80)
+        Speciality = row['Specialist titles']
+
+        MaleFemale = random.randint(0, 1)
+        if MaleFemale == 0:
+            FirstName = names.get_first_name(gender='male')
+
+        if MaleFemale == 1:
+            FirstName = names.get_first_name(gender='female')
         
+        created_physician=[[FirstName,LastName,str(LicenseNo),HireDate,Age,Speciality]]
+        created_physician_df = pd.DataFrame(created_physician,columns=['FirstName','LastName','LicenseNo','HireDate','Age','Specialty'])
+        concated_physician = pd.concat([concated_physician,created_physician_df])
```

Comment:
there's something wrong going with those data frames... can't you create concated_physitian in the beginning with all necessary columns and just insert new data from created_physician on the fly?